packages/js/product-editor/src/components/advice-card/style.scss

```diff
@@ -2,40 +2,44 @@ $card-border-color: $gray-400;
 $card-border-width: 2px;
 
 .woocommerce-advice-card {
-	&.components-card {
-		box-shadow: none;
-		background-image: url( "data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23" + str-slice( '#{ $card-border-color }', 2 ) + "' stroke-width='#{ $card-border-width }' stroke-dasharray='4 4' stroke-dashoffset='46' stroke-linecap='butt'/%3e%3c/svg%3e" );
-		border-radius: 4px;
+	background-image: url( "data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23" + str-slice( '#{ $card-border-color }', 2 ) + "' stroke-width='#{ $card-border-width }' stroke-dasharray='4 4' stroke-dashoffset='46' stroke-linecap='butt'/%3e%3c/svg%3e" );
```

Comment:
Not sure why we need this here?

Reply:
The dotted style is more appropriate for the design as the regular style is not accurate.

Products in this group | Upsells
--------|----------
<img width="671" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/410a1695-4927-404e-b513-aa803519bbf7"> | <img width="684" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/f68086be-8647-43e4-82ba-5c06030b5a35">
 
btw I was considering to add a prop to set the dotted style, too